doc/source/whatsnew/v0.20.0.txt

```diff
@@ -557,6 +557,7 @@ Other Enhancements
 - The ``display.show_dimensions`` option can now also be used to specify
   whether the length of a ``Series`` should be shown in its repr (:issue:`7117`).
 - ``parallel_coordinates()`` has gained a ``sort_labels`` keyword arg that sorts class labels and the colours assigned to them (:issue:`15908`)
+- ``loc`` indexer with datetime strings now works correctly for length 1 DataFrames and is consistent with the behaviour of using a ``datetime.datetime`` object (:issue:`16071')
```

Comment:
move to Bug / Conversion section

just say: ``Bug in indexing with partial string indexing with a len-1 ``DataFrame`` ``